app/models/spotlight/resources/searchworks.rb

```diff
@@ -1,15 +1,14 @@
 module Spotlight::Resources
+  # Resource provider for searchworks pages
   class Searchworks < Spotlight::Resources::DorResource
-
     self.weight = -1000
 
-    def self.can_provide? res
-      !!(res.url =~ /^https?:\/\/searchworks[^\.]*.stanford.edu/)
+    def self.can_provide?(res)
+      res.url.match(%r{^https?://searchworks[^\.]*.stanford.edu/}).present?
     end
 
     def doc_id
-      url.match(/^https?:\/\/searchworks[^\.]*.stanford.edu\/.*view\/([^\/\.#]+)/)[1]
+      url.match(%r{^https?://searchworks[^\.]*.stanford.edu/.*view/([^/\.#]+)})[1]
```

Comment:
In this case, I've come around to the rubocop convention for using `%r` when the regular expression has embedded slashes. It is so much easier to reason about escaping (even going back through working code, I was briefly surprised about the escaping we had to do in `[^\/\.#]`.